GoogleDataLogger/GoogleDataLogger/Classes/GDLLogEvent.m

```diff
@@ -29,4 +29,36 @@ - (instancetype)initWithLogMapID:(NSString *)logMapID logTarget:(NSInteger)logTa
   return self;
 }
 
+#pragma mark - NSSecureCoding and NSCoding Protocols
+
++ (BOOL)supportsSecureCoding {
+  return YES;
+}
+
+- (id)initWithCoder:(NSCoder *)aDecoder {
+  NSString *logMapID = [aDecoder decodeObjectOfClass:[NSObject class] forKey:@"_logMapID"];
```

Comment:
It might be good to have constants here for the keys since they're being used in two places.

Reply:
Done.